api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -133,6 +170,14 @@
     return Audiences.COLLECTOR;
   }
 
+  /**
+   * Gets the audience type.
+   *
+   * @return the audience type
+   * @since 4.8.0
+   */
+  @NonNull Key audienceType();
```

Comment:
Isn't this a breaking change for Audience implementors? Audience does not have `ApiStatus.NonExtendable` on it.

Reply:
I think defaulting to an `UNKNOWN` type is a good solution for this.